app/assets/stylesheets/viewSubmission.css

```diff
@@ -1,116 +1,117 @@
 #codeTable {
-	border-collapse: collapse;
+  border-collapse: collapse;
 }
 
 tr.filename {
-	background-color: #ccc;
+  background-color: #ccc;
 }
 
 .line {
-	cursor:default;
+  cursor:default;
 }
 
 .line td{
-	padding-right: 10px;
-	padding-left: 10px;
-	font-family: monospace;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-family: monospace;
 }
 
 .lineNumber {
-	text-align: right;
+  text-align: right;
 }
 
 /* make the 'X' big and red */
 .line .deleteAnnotation {
-	color: #F00;
-	font-size: 18px;
+  color: #F00;
+  font-size: 18px;
 }
 
 .line .createAnnotation,
 .line .annotationModifier {
-	opacity: 0.0;
+  opacity: 0.0;
 }
 
 .line:hover,
 .line:hover .annotation,
 .line:hover .annotationModifier {
-	/* annotations show up on mouseover */
-	background-color: #DDD;
-	opacity: 1.0;
-	cursor: pointer;
+  /* annotations show up on mouseover */
+  background-color: #DDD;
+  opacity: 1.0;
+  cursor: pointer;
 }
 
 .line.annotated .annotation,
 .line.annotating .annotation, 
 .line.annotating .annotationModifier {
-	opacity: 1.0;
+  opacity: 1.0;
 
-    /* spice up the annotations a little to make them more obvious */
-    font-size: 13px;
-    font-weight: bold;
-    /*font-style: italic; */
+  /* spice up the annotations a little to make them more obvious */
+  font-size: 13px;
+  font-weight: bold;
+  /*font-style: italic; */
 }
 
 .line a:link {
-	color: black;
-	text-decoration: none;
-	font-family: monospace;
+  color: black;
+  text-decoration: none;
+  font-family: monospace;
 }
 
 .code {
-	white-space: pre-wrap;
+  padding: 2px .5em;
+  white-space: pre-wrap;
 }
 
 .divCode,
 .line .annotation,
 .descript {
-	word-wrap:break-word;
+  word-wrap:break-word;
 }
 
 .divCode {
-	max-width: 560px;
+  max-width: 81em; /* 80 characters plus padding */
```

Comment:
Here's important change # 2. (81 because there is a code tag under each divCode (maybe unnecessary?), and each code tag has .5em padding on the left and right sides.